### ISSUE TITLE
How many log files to keep archived setting is set to 0

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#14889]
# Bug Report

## Description

How many log files to keep archived setting is set to 0

## Steps to reproduce

1. install the last 2.29 / master from scratch
2. open advanced and check the logs item

#### Expected behavior

Should be defaulted to `1`

#### Actual behavior

<img width="1312" alt="Screenshot 2024-05-23 at 12 47 27" src="https://github.com/status-im/status-desktop/assets/82375995/5a60e143-fdd1-4a61-a5f6-9be056c3fe3f">

<img width="1312" alt="Screenshot 2024-05-23 at 12 47 30" src="https://github.com/status-im/status-desktop/assets/82375995/bf72c95d-80c5-426a-97df-c2e162dee1b4">

### Additional Information

- Status desktop version: 2.29 or master from may-23-2024
- Operating System: macOS Sonoma 14.4.1

**Comment by caybro** [comment]
For new user, this is coming back as `0`. As part of testiing, I also tried changing it to `2` but it results in:

![image](https://github.com/status-im/status-desktop/assets/5377645/d96ab76a-3a50-4813-8a22-e7feb1cdc841)


Meaning I can no longer log in :/

Console output:
```
DBG 2024-05-23 12:52:12.528+02:00 app state changed                          tid=567346 file=view.nim:130 state=AppLoadingState
DBG 2024-05-23 12:52:12.537+02:00 database reencryption scheduled            topics="accounts-service" tid=567346 file=service.nim:693
DBG 2024-05-23 12:52:12.537+02:00 [threadpool task thread] initiating task   topics="task-threadpool" tid=567510 file=threadpool.nim:54 messageType=TimerTaskArg:ObjectType threadid=567510 task="{\"$type\":\"TimerTaskArg:ObjectType\",\"timeoutInMilliseconds\":1000,\"reason\":\"\",\"vptr\":35714448,\"slot\":\"onWaitForReencryptionTimeout\",\"tptr\":9988080}"
DBG 2024-05-23 12:52:13.539+02:00 starting database reencryption             topics="accounts-service" tid=567346 file=service.nim:716
failed to set `journal_mode` pragma: file is not a database
DBG 2024-05-23 12:52:13.804+02:00 account logged in                          topics="accounts-service" tid=567346 file=service.nim:677
ERR 2024-05-23 12:52:14.340+02:00 node.crashed                               topics="signals-manager" tid=567346 file=signals_manager.nim:58 error="{\"type\":\"node.crashed\",\"event\":{\"error\":\"TorrentConfig.DataDir and TorrentConfig.TorrentDir cannot be \\\"\\\"\"}}"
ERR 2024-05-23 12:52:14.340+02:00 node.login                                 topics="signals-manager" tid=567346 file=signals_manager.nim:54 error="TorrentConfig.DataDir and TorrentConfig.TorrentDir cannot be \"\""
DBG 2024-05-23 12:52:14.340+02:00 app state changed                          tid=567346 file=view.nim:130 state=StartupState
ERR 2024-05-23 12:52:14.342+02:00 login error                                topics="app-controller" tid=567346 file=module.nim:419 methodName=onNodeLogin errDesription="TorrentConfig.DataDir and TorrentConfig.TorrentDir cannot be \"\""
```

**Comment by friofry** [comment]
I think it will be fixed after [#12918](https://github.com/status-im/status-desktop/issues/12918)
(currently I'm working on this)

**Comment by caybro** [comment]
> I think it will be fixed after [#12918](https://github.com/status-im/status-desktop/issues/12918) because setMaxLogBackups may corrupt the NodeConfig
> 
> (currently I'm working on this)

Great, this is what I ran into above I think
